app/models/collection_version.rb

```diff
@@ -40,6 +40,13 @@ def accessioned?
       DepositCollectionJob.perform_later(collection_version)
     end
 
+    after_transition on: :update_metadata, new: :first_draft do |collection_version|
+      if Settings.notify_admin_list
```

Comment:
I added a setting, defaulting to false, to turn on these emails. PR https://github.com/sul-dlss/shared_configs/pull/1834 turns it on in prod (and is harmless otherwise).